deal.II-toolchain/platforms/supported/debian.platform

```diff
@@ -7,12 +7,12 @@
 #   openmpi-bin openmpi-common libopenmpi-dev cmake subversion git \
 #   libblas-dev liblapack-dev libblas3 liblapack3 splint tcl tcl-dev \
 #   environment-modules libsuitesparse-dev libtool libboost-all-dev \
-#   qt4-dev-tools
+#   qt4-dev-tools libgmp-dev
 #
 # Then reboot and run candi again.
 ##
 
-# on debian 9 the candi installed parmetis 4.0.3 is not recognized correctly
+# on Debian the candi installed parmetis 4.0.3 is not recognized correctly
```

Comment:
On (capitalized beginning of sentence ... sorry)

Reply:
Sorry, I just missed that.